src/app/components/Header/Header.tsx

```diff
@@ -16,8 +16,8 @@ export const Header = () => {
 
   return (
     <nav>
-      <div className="bg-base-300 shadow-sm">
-        <div className="container mx-auto flex w-full items-center justify-between gap-4 p-6 pb-4 md:pb-6">
+      <section className="bg-primary-main dark:bg-[#181818] h-[300px] mb-[-188px]">
```

Comment:
@tong Don't use `dark:` modifier and don't try to implement dark mode atm. We gonna reuse the same color variable names by using different color palette. For better understanding check this lib https://www.npmjs.com/package/tw-colors

Reply:
That's cool. I will remove the `dark:` modifier and ignore dark theme for now.